assets/js/components/Dialog.js

```diff
@@ -109,9 +111,14 @@ const Dialog = ( {
 								<Button
 									onClick={ handleConfirm }
 									danger={ danger }
+									disabled={ showSpinner }
 								>
 									{ confirmButton ? confirmButton : __( 'Disconnect', 'google-site-kit' ) }
 								</Button>
+								<Spinner
+									className="googlesitekit-margin-right-auto"
+									isSaving={ showSpinner }
+								/>
 								<Link
 									className="mdc-dialog__cancel-button"
```

Comment:
How about we put the margin on the cancel button (as a `left-auto`) instead since it will always be the last thing. Then in this case we shouldn't need to update `Spinner` to support an additional `className` either.

Reply:
Great idea, I hadn't thought of that 👍